src/api/api.tsx

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios';
+
+import {GoodsData,ThemeData } from '@/types';
+
+const API_URL = axios.create({
+    baseURL: 'https://react-gift-mock-api-leedyun.vercel.app/',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+
+export const fetchThemes = async (): Promise<ThemeData[]> => {
+    try {
+      const response = await API_URL.get<{themes : ThemeData[]}>('api/v1/themes');
+      if (response.status === 200 && response.data && response.data.themes) {
```

Comment:
데이터가 없으면 어차피 빈 배열로 올테니 불피요한 조건이 추가될 필요는 없습니다.

```suggestion
      if (response.status === 200) {
```